torchvision/transforms/v2/_auto_augment.py

```diff
@@ -137,26 +202,39 @@ def _apply_image_or_video_transform(
                 fill=fill_,
             )
         elif transform_id == "Rotate":
-            return F.rotate(image, angle=magnitude, interpolation=interpolation, fill=fill_)
+            return F.rotate(inpt, angle=magnitude, interpolation=interpolation, fill=fill_)
         elif transform_id == "Brightness":
-            return F.adjust_brightness(image, brightness_factor=1.0 + magnitude)
+            return F.adjust_brightness(inpt, brightness_factor=1.0 + magnitude)
         elif transform_id == "Color":
-            return F.adjust_saturation(image, saturation_factor=1.0 + magnitude)
+            return F.adjust_saturation(inpt, saturation_factor=1.0 + magnitude)
         elif transform_id == "Contrast":
-            return F.adjust_contrast(image, contrast_factor=1.0 + magnitude)
+            return F.adjust_contrast(inpt, contrast_factor=1.0 + magnitude)
         elif transform_id == "Sharpness":
-            return F.adjust_sharpness(image, sharpness_factor=1.0 + magnitude)
+            return F.adjust_sharpness(inpt, sharpness_factor=1.0 + magnitude)
         elif transform_id == "Posterize":
-            return F.posterize(image, bits=int(magnitude))
+            return F.posterize(inpt, bits=int(magnitude))
         elif transform_id == "Solarize":
-            bound = _FT._max_value(image.dtype) if isinstance(image, torch.Tensor) else 255.0
-            return F.solarize(image, threshold=bound * magnitude)
+            if check_type(inpt, (datapoints.Image, is_simple_tensor, datapoints.Video)):
+                bound = _FT._max_value(inpt.dtype)
+            else:
+                bound = 255
+            return F.solarize(inpt, threshold=bound * magnitude)
         elif transform_id == "AutoContrast":
-            return F.autocontrast(image)
+            return F.autocontrast(inpt)
         elif transform_id == "Equalize":
-            return F.equalize(image)
+            return F.equalize(inpt)
         elif transform_id == "Invert":
-            return F.invert(image)
+            return F.invert(inpt)
+        elif transform_id == "Flip":
+            return F.horizontal_flip(inpt)
+        elif isinstance(inpt, datapoints.BoundingBox):
+            return inpt
```

Comment:
This `isinstance()` condition is not consistent with other `transform_id` conditions and makes code more difficult to understand.

Reply:
This condition is to return original bbox for all following `"SolarizeAdd", "Cutout", "BBox_Cutout"` transforms

Reply:
Let's remove it `isinstance(inpt, datapoints.BoundingBox)` and all AAdet related "SolarizeAdd", "Cutout", "BBox_Cutout" transforms from here. We can override `_AutoAugmentDetectionBase._apply_transform` where we can add these special cases.